packages/@ember/-internals/glimmer/lib/modifiers/action.ts

```diff
@@ -200,6 +200,10 @@ class ActionModifierManager implements InternalModifierManager<ActionState, obje
     return new ActionState(element, owner, actionId, actionArgs, named, positional);
   }
 
+  getDebugInstance(state: ActionState): unknown {
+    return state;
+  }
```

Comment:
I think we talked about this and we want to return `null`? We don't really want to leak the internal state bucket, and it's not really useful for it to show up on the inspector

Reply:
right! updated